sim/battle.ts

```diff
@@ -2245,6 +2245,7 @@ export class Battle {
 					this.runEvent('BeforeFaint', pokemon, faintData.source, faintData.effect)) {
 				this.add('faint', pokemon);
 				if (pokemon.side.pokemonLeft) pokemon.side.pokemonLeft--;
+				if (pokemon.side.pokemonLeft === 0) this.send('sideupdate', `${pokemon.side.id}\n|request|null`);
```

Comment:
Is this the right place for this? Shouldn't it be in the same place as the other updates?